### PR TITLE
Add 'Get help' info

### PR DESCRIPTION
**Pull request by David-Feetenby**
Add support information to footer in the teacher interface
Screenshot 2022-12-15 at 15 41 21

### CODE REVIEW

**Comment by andrewscrivener** [comment]
Looks good to me @David-Feetenby 👍

File: app/views/layouts/base.html.erb [hunk 1]
```diff
@@ -52,11 +52,22 @@
...
+        meta_items: {
+          'Accessibility': '/accessibility',
+          'Cookies': '/cookies',
+          'Privacy': '/privacy'
+        } ) do |footer| %>
+      <% footer.navigation do %>
+        <% if current_teacher %>
+          <div class="govuk-footer__section">
+            <h3>Get help</h3>
+            <p>If you're having difficulty completing your application form, contact: <a class="govuk-link"
```
Comment (gpeng) [inline app/views/layouts/base.html.erb:65]:
```suggestion
            <p>If you‘re having difficulty completing your application form, contact: <a class="govuk-link"
```

Also on line 67 in this file and 4 and 5 in the show view.

**Review by gpeng — CHANGES_REQUESTED** [review]

File: app/views/layouts/base.html.erb (hunk 1 above)
Comment (David-Feetenby) [reply app/views/layouts/base.html.erb]: Done 👍 
Removed the content_for from the show view as we aren't using 'yield' anymore so it is no longer necessary

**Comment by gpeng** [comment]
Screenshot 2022-12-15 at 14 49 41

Is that the correct alignment of the content? Shouldn't it be the same width as the stuff underneath?

**Comment by David-Feetenby** [comment]
> Screenshot 2022-12-15 at 14 49 41
> 
> Is that the correct alignment of the content? Shouldn't it be the same width as the stuff underneath?

They should be aligned now

File: app/views/layouts/base.html.erb [hunk 2]
```diff
@@ -52,11 +52,22 @@
...
+          'Cookies': '/cookies',
+          'Privacy': '/privacy'
+        } ) do |footer| %>
+      <% footer.navigation do %>
+        <% if current_teacher %>
+          <div class="govuk-footer__section">
+            <h3 style = "margin-left: 100px">Get help</h3>
+            <p>If you‘re having difficulty completing your application form, contact: <a class="govuk-link"
+            href="mailto:<%= t('service.email') %>"><%= t('service.email') %></a> for support.</p>
+            <p>We'll aim to respond to you within 2 working days.</p>
```
Comment (gpeng) [inline app/views/layouts/base.html.erb:67]:
```suggestion
            <p>We‘ll aim to respond to you within 2 working days.</p>
```

There's another one here that needs changing too.

**Review by gpeng — APPROVED** [review]

**Pull request merged**